apps/web-giddh/src/app/userDetails/components/subscription/subscription.component.ts

```diff
@@ -49,6 +49,18 @@ export class SubscriptionComponent implements OnInit, OnDestroy, OnChanges {
     @ViewChild(MatAccordion) accordion: MatAccordion;
     /** This will use for move company in to another company  */
     @ViewChild("moveCompany", { static: false }) public moveCompany: any;
+    /** This will change the search bar width dynamically */
+    public menuOneWidth: Number = 4;
```

Comment:
Consider using primitive type `number` in place of `Number` through out this file